.github/workflows/lint.yml

```diff
@@ -0,0 +1,34 @@
+name: Test
+
+on: [push]
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.10.6]
+        redis-version: [6]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Start Redis
+      uses: supercharge/redis-github-action@1.2.0
+      with:
+        redis-version: ${{ matrix.redis-version }}
```

Comment:
Sorry for only realising this now, but I don't see anywhere that we're using Redis in the config?